controllers/userControllers.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const User = require("../models/User");
+
+async function registerUser(req, res) {
+  try {
+    const payload = req.body;
+    console.log(payload);
+    if (!payload) {
+      res.status(400);
+    } else {
+      res.status(201);
```

Comment:
in case of 201, ideally we can send a message 'created'

Reply:
okay sir